### DISCUSSION TITLE
顶栏的订阅不是真的“订阅”

### DISCUSSION THREAD

**Discussion by actforjason** [the1812/Bilibili-Evolved#4777] [功能建议 / Ideas]
顶栏的订阅是“追番和追剧”，可以填加收藏里的 “我的收藏和订阅” 到顶栏吗，这个是订阅的合集
![图片](https://github.com/the1812/Bilibili-Evolved/assets/57058325/e13d4987-9675-457d-bf7b-36f6127065b7)

**Comment by Geeyun-JY3** [comment]
叫「订阅」是因为 b 站个人空间页对应 tab（URL：`https://space.bilibili.com/<uid>/bangumi`）之前就是这么叫的……
![image](https://github.com/the1812/Bilibili-Evolved/assets/49396421/1b99a323-461d-4b4a-b945-8b3bfd0ad543)

最早也是去年十月份之后才改成现在的这个「追番追剧」。

**Reply by Geeyun-JY3** [reply]
@the1812 或许`自定义顶栏`该跟进改名了，免得用户误会。
